pygame/README.md

```diff
@@ -3,10 +3,17 @@
 This folder contains example code using [pygame](https://github.com/pygame/pygame) to obtain
 camera images and then perform image classification or object detection on the Edge TPU.
 
+This code works on Linux using a webcam, Raspberry Pi with the Pi Camera, and on the Coral Dev Board using a webcam. For the first two, you also need a Coral USB/PCIe/M.2 Accelerator.
+
 ## Set up your device
 
 1.  First, be sure you have completed the [setup instructions for your Coral
-    device](https://coral.ai/docs/setup/).
+    device](https://coral.ai/docs/setup/). If it's been a while, repeat to be sure
+    you have the latest software.
+
+    Importantly, you should have the latest TensorFlow Lite runtime installed
```

Comment:
Might be worth adding "you can check which version is installed with `pip3 freeze`" or similar, i dunno.

Reply:
Would there be a way to do this that wouldn't involve hardcoding the most recent version of tflite-runtime or having the user also go to the [Python quickstart guide](https://www.tensorflow.org/lite/guide/python) (the page where I could clearly find version information)?

Reply:
I don't think you need to find a way to pull the expected version and keep the doc updated, i'm saying just telling them (perhaps a beginner), how to quickly check the version they have ( `pip freeze` ) would make it beginner friendly.

Reply:
```pip3 show tflite_runtime``` will show you the version. Currently we require 1.14 i think